network/stream/delivery.go

```diff
@@ -62,52 +59,42 @@ func NewDelivery(kad *network.Kademlia, netStore *storage.NetStore) *Delivery {
 
 // RetrieveRequestMsg is the protocol msg for chunk retrieve requests
 type RetrieveRequestMsg struct {
-	Addr      storage.Address
-	SkipCheck bool
-	HopCount  uint8
```

Comment:
protocol message change. after we merge this, we need to bump protocol versions.